src/main/java/org/terasology/structureTemplates/util/BlockRegionTransform.java

```diff
@@ -92,7 +92,8 @@ public Region3i transformRegion(Region3i region) {
 
 
     public BlockRegion transformRegion(BlockRegion region) {
-        return new BlockRegion(transformVector3i(region.getMin(new Vector3i())),
+        return new BlockRegion().union(
+            transformVector3i(region.getMin(new Vector3i()))).union(
             transformVector3i(region.getMax(new Vector3i())));
```

Comment:
On a closer look, the solution with `union` actually looks pretty neat if formatted differently :nerd_face: 

```suggestion
        return new BlockRegion()
                .union(transformVector3i(region.getMin(new Vector3i())))
                .union(transformVector3i(region.getMax(new Vector3i())));
```